_data/other.yml

```diff
@@ -343,6 +343,13 @@ websites:
       tfa: Yes
       software: Yes
 
+    - name: Ulule
+      url: https://www.ulule.com/
+      twitter: ulule
+      facebook: ulule
```

Comment:
For me (in FR) the ulule facebook didn't redirect me to the FR one, seem to be better for all.